src/python/gem5/utils/multisim/multisim.py

```diff
@@ -172,8 +176,10 @@ def _run(module_path: Path, id: str) -> None:
     sim_list[0].override_outdir(subdir)
     # This doesn't do anything if none of the redirect options are passed
     override_re_outdir(subdir)
-
-    sim_list[0].run()
+    try:
+        sim_list[0].run()
```

Comment:
Did you test this? Try putting an error in the simulator (e.g., maybe in the exit event handler?) and see what happens.

Reply:
Yes, I raised an exeption in exit event handler and it was caught and the error message was printed.

```
info: Error running simulator fs-sim: Exit event handler not implemented
```

The other simulations were not affected by this. I tested by running this fullsystem simulation with other SE mode simulation. The SE mode simulation ran as expected irrespective if the FS simulation was run first or last.